pkg/provision/providers/qemu/create.go

```diff
@@ -66,7 +77,11 @@ func (p *provisioner) Create(ctx context.Context, request provision.ClusterReque
 
 	fmt.Fprintln(options.LogWriter, "creating load balancer")
 
-	if err = p.CreateLoadBalancer(state, request); err != nil {
+	// TODO: Find a way to reuse provision.BaseNodeRequest filter methods
+	controlplanes := xslices.Filter(request.Nodes, func(n vm.NodeRequest) bool { return n.Type.IsControlPlane() })
```

Comment:
I couldn't think of a way to reuse the [`provision.BaseNodeRequests.ControlPlaneNodes()`](https://github.com/Orzelius/talos/blob/create-cmd-refactor/pkg/provision/request.go#L73) method with the [`vm.NodeRequests`](https://github.com/Orzelius/talos/blob/create-cmd-refactor/pkg/provision/providers/vm/request.go#L84) type. Please let me know if you can think of something to reduce this duplication.